webpack.prod.js

```diff
@@ -5,68 +5,70 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+module.exports = merge(common('production', { mode: 'production', publicPath }), {
+  mode: 'production',
   devtool: 'source-map',
   optimization: {
     minimizer: [
       new TerserJSPlugin({}),
       new CssMinimizerPlugin({
         minimizerOptions: {
           processorOptions: {
-            preset: ['default', { mergeLonghand: false }] // Fixes bug in PF Select component https://github.com/patternfly/patternfly-react/issues/5650#issuecomment-822667560
-          }
-        }
-      })
+            preset: ['default', { mergeLonghand: false }], // Fixes bug in PF Select component https://github.com/patternfly/patternfly-react/issues/5650#issuecomment-822667560
+          },
+        },
+      }),
     ],
   },
   plugins: [
     new CopyPlugin({
       patterns: [
         {
           from: 'config/config.json',
-          to: `config.json`
+          to: `config.json`,
         },
         // Copy the logos for the service, for usage by third parties
         {
-          from: 'static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-White-RGB.png',
-          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-White-RGB.png'
+          from: `${AppServicesUiComponentsPath}/static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-White-RGB.png`,
+          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-White-RGB.png',
         },
         {
-          from: 'static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB.png',
-          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB.png'
+          from: `${AppServicesUiComponentsPath}/static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB.png`,
+          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB.png',
         },
         {
-          from: 'static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.png',
-          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.png'
+          from: `${AppServicesUiComponentsPath}/static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.png`,
+          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.png',
         },
         {
-          from: 'static/images/Logo-Red_Hat-OpenShift-Application_Services-A-Reverse-RGB.svg',
-          to: 'Logo-Red_Hat-OpenShift-Application_Services-A-Reverse-RGB.svg'
+          from: `${AppServicesUiComponentsPath}/static/images/Logo-Red_Hat-OpenShift-Application_Services-A-Reverse-RGB.svg`,
+          to: 'Logo-Red_Hat-OpenShift-Application_Services-A-Reverse-RGB.svg',
         },
         {
-          from: 'static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB-310x117.png',
-          to: 'Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB-310x117.png'
+          from: `${AppServicesUiComponentsPath}/static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Reverse-RGB-310x117.png`,
```

Comment:
This copy will fail at build time if the files can't be found. So if the file names change on `app-services-ui-components`, we will know before going live with a new version.